index.html

```diff
@@ -0,0 +1,806 @@
+      </h1>
+      <div class='dgrid div-margin'>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/4.png " alt="article">
+          <div>
+            <h2>Satoshi Nakaboto: ‘Kazakhstan to pump $700M into crypto mining’</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/5.png" alt="article">
+          <div>
+            <h2>Norwegian ebike owners ride 4x as much after buying their bikes</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/6.png" alt="article">
+          <div>
+            <h2>Why 'micromobility' data is the key to cities reaching their sustainability goals</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/7.jpg" alt="article">
+          <div>
+            <h2>These 10 apps, now on sale, could help change how you do virtually everything </h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/8.png" alt="article">
+          <div>
+            <h2>These new semiconductors could revolutionize the solar energy industry</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/9.png" alt="article">
+          <div>
+            <h2>How Slack employees use Slack</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/10.jpg" alt="article">
+          <div>
+            <h2>Tech dismally failed a 25-year challenge to achieve gender equality by 2020</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/11.jpeg" alt="article">
+          <div>
+            <h2>What are geofence warrants?</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class=" article-section-short article-section-ipad">
```

Comment:
This section is overlapping with `latest news` section between screen width of `1024px` to `1301px`. Please fix this 👇 .

![image](https://user-images.githubusercontent.com/57726348/92636085-6e053b00-f2f4-11ea-948e-360d2df90e5c.png)